src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -41,12 +41,14 @@ public Response buildNoContentResponse() {
   }
 
   public Response buildResponseWithLocation(String okapi, String endpoint, Object body) {
+    logger.debug("buildResponseWithLocation:: Trying to build response with location endpoint: {}, body: {}, for cache okapi: {}", body, endpoint, okapi);
     try {
       return Response.created(new URI(okapi + endpoint))
         .header(CONTENT_TYPE, APPLICATION_JSON)
         .entity(body)
         .build();
     } catch (URISyntaxException e) {
+      logger.error("Failed to build response with location", e);
```

Comment:
Should I use method name beginning of message
`logger.error("buildResponseWithLocation::: Failed to build response with location", e);`

based on documentation they mostly don't use
![image](https://user-images.githubusercontent.com/113523904/201314097-c7e3f54b-ec55-4e13-91f6-5ea6a961fcfb.png)

in warn message they use
![image](https://user-images.githubusercontent.com/113523904/201314160-ddc40987-dedf-45ba-a50d-37687ad67c86.png)

Reply:
For error level logs I think we can omit prefix with method name